packages/@aws-cdk/assertions/README.md

```diff
@@ -144,9 +144,9 @@ The APIs `hasMapping()` and `findMappings()` provide similar functionalities.
 
 ## Special Matchers
```

Comment:
The section in the README for `Full Template Match` should be updated as well since that is no longer a requirement.

Reply:
What needs to be updated specifically?

Reply:
I feel like it shouldn't be called `Full Template Match`. It is now a partial template match. If a user wanted to be sure that there was a full template match, they would have to say

```ts
assert.templateMatches(Match.objectEquals(...));
```

@nija Isn't it a tad bit misleading to say it is a full template match when it is in fact a partial one?